index.html

```diff
@@ -430,9 +430,11 @@ <h4>DataIntegrityProof</h4>
 `controller`.
           </p>
           <p>
-The `proofValue` property of the proof MUST be a detached ECDSA
-produced according to [[FIPS-186-5]], encoded according to [[MULTIBASE]] using
-the base58-btc base encoding.
+The `proofValue` property of the proof MUST be an ECDSA or deterministic ECDSA
+signature produced according to [[FIPS-186-5]] using the curves and hashes as
+specified in section <a href="#algorithms"></a>, encoded according to section 7 
+of [[RFC4754]], and serialized according to [[MULTIBASE]] using the base58-btc 
+base encoding.
```

Comment:
It would be nice to either say that the encoding used is the same one that is used by Web Crypto ... or informally mention the IEEE P1363 format since we don't want to link to an inaccessible document). Something like this may be good enough?

```suggestion
specified in section <a href="#algorithms"></a>, encoded according to section 7 
of [[RFC4754]] (sometimes referred to as the IEEE P1363 format), and serialized
according to [[MULTIBASE]] using the base58-btc base encoding.
```